pkg/netid/netid.go

```diff
@@ -0,0 +1,61 @@
+package netid
+
+/*
+Accessor methods to annotate namespace for multitenant support
+*/
+
+import (
+	"fmt"
+	"math"
+	"strconv"
+
+	kapi "k8s.io/kubernetes/pkg/api"
+)
+
+const (
+	// Current assigned VNID for the namespace
+	VNIDAnnotation string = "pod.network.openshift.io/multitenant.vnid"
+	// Desired VNID for the namespace
+	WantsVNIDAnnotation string = "pod.network.openshift.io/multitenant.wants-vnid"
+)
+
+var (
+	ErrorVNIDNotFound = fmt.Errorf("VNID/WantsVNID annotation not found")
```

Comment:
given that annotation names themselves tend to have slashes in them, it would be better to not use one as a non-name-component in the error message text. ("VNID or WantsVNID" would be a simple fix, but you could also just create the error when you throw it and %s the annotationKey value into it...)